lib/src/widgets/view.dart

```diff
@@ -225,6 +228,35 @@ class EditorJSViewState extends State<EditorJSView> {
                   }
                 }
                 break;
+                case "integration":
+                  log('integration');

```

Comment:
Looks great! Can you please add some robustness checks (if conditions) to make sure we have the data in the correct form when rendering the editorjs block, the above cases have similar stuff